### ISSUE TITLE
[Feature/Enhancement] 

### ISSUE THREAD

**Issue by Iznik285** [dvorka/mindforger#1457]
**Enhancement / feature description:**

It would be useful to be able to select a series of lines of text and when choosing format list it adds the relevant list item at the beginning of each line as opposed to the current method of selecting the format list first and then filling it in as needed.

**Comment by dvorka** [comment]
@Iznik285 good idea! I would use it (e.g. in case of copy pasting text to MindForger) as well. Adding it to the plan. Thanks!
